src/viking.js

```diff
@@ -1,11 +1,90 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack() {
+        return this.strength;
+    }
+    receiveDamage(damage){
+        this.health = this.health -= damage;
+    }
```

Comment:
Phe, no receiveDamage não precisa ter dois this.health aí, só um já está correto :)